src/services/AccountService.ts

```diff
@@ -58,6 +63,26 @@ export class AccountService {
             account.acceptTermsPrivacy = acceptTermsPrivacy || account.acceptTermsPrivacy;
         }
 
+        if (organisation) {
```

Comment:
@GarfDev I'm familiar with this pattern and I also apply it for PATCH operations (but I dont really like it, since it will become a lot of code to manage for large updates).

Isn't there a smarter way to do this with mongoose, where maybe that library filters out the props that we need to update for us? Would `account.updateOne(req.body)` not suffice if we validate the payload properly?